src/geo/map/legends/custom-legend-model.js

```diff
@@ -7,6 +7,24 @@ var CustomLegendModel = StaticLegendModelBase.extend({
       type: 'custom',
       items: []
     });
+  },
+
+  parse: function (r, opts) {
```

Comment:
> there is no data returned by the server

That's exactly what I mean. IMO `parse` is meant to "adapt" responses coming form the server when the model is fetched... But here, models are use differently, since they are not connected to an endpoint and hence there's no need to parse IMO...